inference-engine/src/transformations/include/transformations/serialize.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+// ATTENTION :
```

Comment:
Move to doxygen class comment

Reply:
Done in this [commit](https://github.com/openvinotoolkit/openvino/pull/2579/commits/1be8c85bc06795554e80cf5b9903214b2f2248e7)